samples/parselib/src/main/cpp/CMakeLists.txt

```diff
@@ -42,15 +42,15 @@ add_library( # Sets the name of the library.
 
         # Provides a relative path to your source file(s).
         # io/stream

```

Comment:
**io**/stream?

Reply:
This change remove "io/" from the path but it is still in the comment on line 44.
Should that be "parselib/stream".

Reply:
done.